### ISSUE TITLE
Crash on toolbox item double-click

### ISSUE THREAD

**Issue by ktraunmueller** [ktraunmueller/Compositor#96]
One of the toolbox items (find out which one) causes crashes in `insert(command:precededBy:followedBy:)` in the line 
```
selection = NSRange(location: selection.location + offset, length: selection.length)
```

Crash group: https://rink.hockeyapp.net/manage/apps/425865/app_versions/52/crash_reasons/175823364?type=crashes

**Comment by ktraunmueller** [comment]
Chris writes:
> I did manage to crash the app by creating a new article and then double clicking on the image in the toolbox.

and that seems to produce the exact same stack trace: https://rink.hockeyapp.net/manage/apps/425865/app_versions/53/crash_reasons/177186917?type=crashes

Strangely enough, this does not crash in a debug build. Investigating...

**Comment by ktraunmueller** [comment]
**Fixed** in 0.9.2
